packages/chart/src/Pie.ts

```diff
@@ -17,10 +19,15 @@ export class Pie extends SVGWidget {
 
     protected _totalValue: number;
 
-    labelWidgets;
-    d3Pie;
-    d3Arc;
-    d3LabelArc;
+    private d3Pie;
+    private d3Arc;
+    private d3LabelArc;
```

Comment:
I suspect the d3Pie / Arc and Label should not be modified - I was referung to your new properties specific to the label layouts.